### ISSUE TITLE
Crashes immediately

### ISSUE THREAD

**Issue by geekalert** [utmapp/UTM#2301]
**Describe the issue**
As soon as I touch the UTM app, the screen flashes like it will load, but it never does — just crashes back to the desktop.

**Configuration (required)**
* UTM Version: v2.0.20
* OS Version: iPadOS 14.3
* Device Model: iPad Pro 2018 12.9”
* Is it jailbroken (name jailbreak used)? No
* How did you install UTM? AltStore

**Crash log**
There are no crash logs with today’s date for UTM, so I can put nothing here. This is a stock, non-jailbroken 2018 iPad Pro 12.9”. There is literally nothing but stock on it, AltStore, and UTM.

**Debug log**
Since UTM crashes immediately when launched, I cannot provide this. However, I have tried uninstalling UTM, uninstalling AltStore, reinstalling AltStore, and reinstalling the latest version of AltStore and UTM. Nothing changes. It worked fine on iPadOS 14.3 on this exact device in December. 

**Upload VM**
Since I cannot make a VM, I cannot provide this. I was hoping to come here and find a glaring post about an issue. I was bummed not to find anything. I had hoped the latest version would be the fix, but still no joy.

**Comment by osy** [comment]
Is this relates to the recent AltStore issues? I.e. does installing anything else work?

**Comment by geekalert** [comment]
Hey, good thinking. I haven’t tried any other apps, since I literally got AltStore for UTM. What is another app I can try to sideload to see if it has the same problem?

**Comment by osy** [comment]
Try anything listed in altstore. Like delta.

**Comment by geekalert** [comment]
Sorry it took me so long to get back to this issue. I was *not* able to install anything new. So I went to the AltStore issues and saw the update. Simply letting the app update itself, however, did not work. I am on Win 10, so I needed to completely remove the old version of AltStore from my iPad, AltServer from my system, reinstall latest beta of AltStore for Win, and reinstall AltStore on the iPad. Then I was able to successfully load UTM. Thanks for the kick in the right direction -- appreciate it! *runs to build happy little development playgrounds on the iPad Pro* Hooray!
